docker/package/packages.py

```diff
@@ -506,78 +506,74 @@ def mk_node_unit(
 )
 
 
-def mk_rollup_packages(*protos):
-    def mk_units(proto):
-        startup_script = f"/usr/bin/tezos-sc-rollup-node-{proto}-start"
-        service_file = ServiceFile(
-            Unit(after=["network.target"], description=f"Tezos smart rollup node"),
-            Service(
-                environment_files=[f"/etc/default/tezos-smart-rollup-node-{proto}"],
-                environment=[f"PROTOCOL={proto}"],
-                exec_start_pre=[
-                    "+/usr/bin/setfacl -m u:tezos:rwx /run/systemd/ask-password"
-                ],
-                exec_start=startup_script,
-                exec_stop_post=[
-                    "+/usr/bin/setfacl -x u:tezos /run/systemd/ask-password"
-                ],
-                state_directory="tezos",
-                user="tezos",
-                type_="forking",
```

Comment:
Could you please elaborate a bit on this `type_="forking"` -> `type_="simple"` change? I didn't get it, to be honest 😅

Reply:
oh, I guess `forking` is about having different services like we do for `tezos-node`, but since we have single smart rollup node binary, it isn't needed now 🤔

Reply:
`octez-smart-rollup-node` does not seem to fork. `forking` systemd service consider service activated when it forks into its daemon. It is the case for `octez-node`. But `octez-smart-rollup-node` just runs and service hangs into `activating` state, failing in the end because of the timeout.

Reply:
`forking` systemd service is about having `fork()` system call during service startup

Reply:
Got it, thanks for the clarification